java/build.gradle

```diff
@@ -41,6 +39,8 @@ subprojects {
         options.deprecation = true
     }
 
+    // TODO do we need this plug-in?
+    apply plugin: 'idea'
```

Comment:
Not clear if we need this idea plug-in at all. If not it would be better to remove it.

Reply:
https://docs.gradle.org/current/userguide/idea_plugin.html

If I understand what this does correctly, I don't think it's necessary. We never use this editor